src/toml-generic_parse.adb

```diff
@@ -2051,9 +2063,13 @@ is
 
          --  Then the exponent absolute value
 
-         if not Read_Simple_Integer (Exponent) then
-            return False;
-         end if;
+         declare
+            Junk : Natural;
```

Comment:
Let’s follow the same convention as in the lest of the codebase: `Dummy_Leading_Zeros : Natural;`